Makefile

```
@@ -0,0 +1,92 @@
+	@echo "  make test-clean              - Clean test cache and run tests"
+	@echo "  make build-job               - Build the job application binary"
+	@echo "  make install-pre-push-hook   - Install the pre-push git hook"
+	@echo "  make uninstall-pre-push-hook - Uninstall the pre-push git hook"
+
+job:
+	go run cmd/job/main.go
+
+lint:
+	golangci-lint run ./...
+
+test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (6).

(maxbodylength)